kore/src/Kore/Step/Simplification/CeilSimplifier.hs

```diff
@@ -10,40 +10,98 @@ module Kore.Step.Simplification.CeilSimplifier
     , runCeilSimplifierWith
     ) where
 
-import Prelude.Kore
+import Prelude.Kore hiding
+    ( (.)
+    )
 
+import Control.Category
+    ( Category (..)
+    )
+import Control.Error
+    ( MaybeT
+    )
+import Control.Monad
+    ( (<=<)
+    )
+import qualified Control.Monad.Morph as Morph
 import Control.Monad.Reader
     ( ReaderT
     , runReaderT
     )
 import Data.Profunctor
     ( Profunctor (..)
     )
+import Data.Profunctor.Choice
+    ( Choice (..)
+    )
 
 import Kore.Sort
 import Kore.Syntax.Ceil
 
 newtype CeilSimplifier simplifier input output =
-    CeilSimplifier { runCeilSimplifier :: Ceil Sort input -> simplifier output }
+    CeilSimplifier {
+        runCeilSimplifier :: Ceil Sort input -> MaybeT simplifier output
+    }
```

Comment:
Any reason this formatting is prefered here to putting `{` on the line with `runCeilSimplifier ...`?